src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+	<www-page id="borrower-profile">
+		<div class="container">
+			<div class="grid grid-cols-12 gap-x-2">
+				<div class="col-span-12 md:col-start-2 md:col-span-10 lg:col-span-7"
+					style="background-color:lightgreen;"
```

Comment:
are these just for debugging + temporary grid visibility?

Reply:
I put them there, so you can see the grid breaking across screen sizes if you pulled it up in storybook. I'll remove the inline style before merging.